torch/_higher_order_ops/flex_attention.py

```diff
@@ -1146,17 +1146,47 @@ def flex_attention_backward_fake_tensor_mode(
     torch.Tensor, torch.Tensor, torch.Tensor, tuple[Optional[torch.Tensor], ...]
 ]:
     with mode:
-        grad_query = torch.empty_like(query)
```

Comment:
This looks like https://github.com/pytorch/pytorch/commit/a3e0d4bf0797f862479c2f38d90cebd151449418, does that one also need to be empty_strided?

Reply:
Good call out!